### DISCUSSION TITLE
Nerd Foont Symbol

### DISCUSSION THREAD

**Discussion by lxg208** [JanDeDobbeleer/oh-my-posh#4546] [Troubleshoot]
I'm trying to put 󰌘 (nf-md-lan_connect f0318) in the prompt. It works if I put the symbol 󰌘 in the theme file. If I put \uf0318 in the file, then it's displayed as 8. Is there a way I can use \uf0318 style? The reason is if I put the actual symbol, I cannot see it for editors without Nerd Font. 

Thanks.

**Comment by JanDeDobbeleer** [comment]
You need to use `\udb80\udf18` as this is a surrogate pair. When using the Nerd Font cheat sheet, you can click on UTF to get the correct notation.
